src/index.html

```diff
@@ -47,149 +47,48 @@
-                xmlns="http://www.w3.org/2000/svg"
-              >
-                <title>Menu</title>
-                <path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z" />
-              </svg>
-            </button>
-          </div>
-          <div
-            class="w-full flex-grow lg:flex lg:items-center justify-end lg:w-auto hidden mt-2 lg:mt-0 bg-white lg:bg-transparent text-black p-4 lg:p-0 z-20"
-            id="nav-content"
+          <span class="text-base sm:hidden" data-i18n="lang_abbr">FR</span
+          ><span class="sr-only sm:not-sr-only" data-i18n="lang_button"
+            >Français</span
           >
-            <input type="checkbox" class="checkbox" id="checkbox" data-testid="darkmode_switch"/>
-            <label
-              for="checkbox"
-              class="checkbox-label mr-4 focus:outline focus:outline-2 focus:outline-red-500 transition"
-            >
-              <i class="text-white ph-fill ph-moon"></i>
-              <i class="text-white ph-fill ph-sun"></i>
-              <span class="ball"></span>
-            </label>
-            <button
-              id="navAction"
-              class="mx-auto lg:mx-0 hover:underline bg-white text-gray-800 font-bold rounded-full mt-4 lg:mt-0 py-4 px-8 shadow-lg opacity-75 transform transition hover:scale-105 duration-300 ease-in-out"
-              data-testid="lang_button"
-            >
-              <span data-i18n="lang_button">Français</span>
-            </button>
-          </div>
-        </div>
+        </button>
       </div>
     </nav>
     <!--Hero-->
-    <div class="pt-24">
-      <div
-        class="container max-w-5xl mb-12 px-3 mx-auto flex flex-wrap flex-row items-center"
-      >
-        <!--Left Col-->
-        <div
-          class="flex flex-col w-full mt-16 mb-6 justify-center items-center text-left"
-        >
-          <div
-            class="w-full max-w-md p-4 bg-white border border-gray-200 rounded-lg shadow-xl sm:p-8 dark:bg-gray-800 dark:border-gray-700"
-            data-testid="component_status_table"
-          >
-            <div class="flex items-center justify-between mb-4">
-              <h5
-                class="text-xl font-bold leading-none text-gray-900 dark:text-white ml-2"
-              >
-                Component
-              </h5>
-              <p class="font-medium text-black dark:text-white mr-2">Status</p>
-            </div>
-            <div class="flow-root">
-              <ul
-                role="list"
-                class="divide-y divide-gray-200 dark:divide-gray-700"
-                id="component_status_list"
-              ></ul>
-            </div>
-          </div>
-          <!-- <img src="../dist/img/component-status-table.png" width="818px" /> -->
-        </div>
+    <div class="bg-gray-200 dark:bg-gray-800">
+      <div class="container mx-auto px-gutterHalf py-gutterHalf w-full">
+        <ul role="list" id="component_status_list" data-testid="component_status_table"></ul>
```

Comment:
I feel like the status table is a bit stretched on desktop.  What about adding some white space in this case so the table is easier to scan visually?
```suggestion
        <ul role="list" class="lg:mx-48 xl:mx-72" id="component_status_list" data-testid="component_status_table"></ul>
```

![image](https://github.com/cds-snc/notification-system-status-frontend/assets/823749/50ab289a-6d65-4229-ab48-ded2207f6c0a)

Reply:
I agree. I think we could 2 more things:
- I removed the larger breakpoints, so that we don't go over 1024. 
- We could also create a grid like in the image below

I want to avoid having layouts with different widths, and have thing align to the left as much as possible. So if the grid layout seems fine, I'd prefer it, but otherwise we can at least reduce the max breakpoint, so that we minimize the difference between center aligned and left aligned. 

<img width="1096" alt="image" src="https://github.com/cds-snc/notification-system-status-frontend/assets/5230720/9812bce6-f15b-45de-8c88-45834b77de46">

Reply:
Visually, I like this better.  It may be slightly harder to scan for problems when it isn't in a stacked list, but I think it looks better tight like this with the component closer to the status rather than stretched out.

Reply:
We may want to consider switching to 2 cols at `lg` instead of `md`.  Here is how md looks for me:
![image](https://github.com/cds-snc/notification-system-status-frontend/assets/823749/df16cacb-11c5-48a4-b5be-a797b84ef746)

Reply:
Good catch! Flexbox being a little too flexible here... I think we can tweak flex wrap on the status too.